docs/cli/claims.md

```diff
@@ -0,0 +1,296 @@
+
+- `actor`
+- `operator`
+- `account`
+- `provider`
+
+### `actor`
+Generate a signed JWT for an actor
+
+#### Usage
+```
+wash claims token actor --name=example -k 
+```
+
+#### Options
+
+`--keyvalue` (Alias `-k`) Enable the Key/Value Store standard capability
+
+`--output` (Alias `-o`) Specify output format (text or json) [default: text]
+
+`--experimental` Whether or not to enable experimental features [env: WASH_EXPERIMENTAL=]
+
+`--msg` (Alias `-g`) Enable the Message broker standard capability
+
+`--http_server` (Alias `-q`) Enable the HTTP server standard capability
+
+`--http_client` Enable the HTTP client standard capability
+
+`--blob_store` (Alias `-f`) Enable access to the blob store capability
+
+`--extras` (Alias `-z`) Enable access to the extras functionality (random nos, guids, etc)
+
+`--logging` (Alias `-l`) Enable access to logging capability
+
+`--events` (Alias `-e`) Enable access to an append-only event stream provider
+
+`--name` (Alias `-n`) A human-readable, descriptive name for the token
+
+`--cap` (Alias `-c`) Add custom capabilities
+
+`--tag` (Alias `-t`) A list of arbitrary tags to be embedded in the token
+
+`--prov` (Alias `-p`) Indicates whether the signed module is a capability provider instead of an actor (the default is actor)
+
+`--rev` (Alias `-r`) Revision number
+
+`--ver` (Alias `-v`) Human-readable version string
+
+`--call-alias` (Alias `-a`) Developer or human friendly unique alias used for invoking an actor, consisting of lowercase alphanumeric characters, underscores '_' and slashes '/'
+
+`--issuer` (Alias `-i`) Path to issuer seed key (account). If this flag is not provided, the will be sourced from $WASH_KEYS ($HOME/.wash/keys) or generated for you if it cannot be found [env: WASH_ISSUER_KEY]
+
+`--subject` (Alias `-s`) Path to subject seed key (module). If this flag is not provided, the will be sourced from $WASH_KEYS ($HOME/.wash/keys) or generated for you if it cannot be found [env: WASH_SUBJECT_KEY]
+
+`--directory` Location of key files for signing. Defaults to $WASH_KEYS ($HOME/.wash/keys) [env: WASH_KEYS]
+
+`--expires` (Alias `-x`) Indicates the token expires in the given amount of days. If this option is left off, the token will never expire
+
+`--nbf` (Alias `-b`) Not before days. Period in days that must elapse before this token is valid. If this option is left off, the token will be valid immediately
+
+`--disable-keygen` Disables autogeneration of keys if seed(s) are not provided
+
+
+### `operator`
+Generate a signed JWT for an operator
+
+#### Usage
+```
+wash claims token operator --name=example
+```
+
+#### Options
+`--name` (Alias `-n`) A human-readable, descriptive name for the token
+
+`--output` (Alias `-o`) Specify output format (text or json) [default: text]
```

Comment:
Super nit: A lot of these flags are global to all the subcommands. It might make it more readable and easier to maintain if we pull those out into their own section

Reply:
Afaik the only global flags are `--output`, `--help` and `--experimental`. We already make a note about `--help` on the main `index.md` page for wash. But the other two flags cannot be used without subcommands so they don't exactly fit in there.